website/components/_app.tsx

```diff
@@ -25,9 +25,17 @@ declare global {
   }
 }
 
+const getCanaonicalUrl = (path: string) => {
+  if (process.env.NEXT_PUBLIC_VERCEL_ENV !== "production") {
```

Comment:
Will `NEXT_PUBLIC_VERCEL_ENV` be set to `production` when we deploy to a custom domain, like `v1` or `docs`?

Reply:
I think it's set to `production` on any build than runs on the default branch:
![image](https://user-images.githubusercontent.com/26760571/224042016-fb6be190-dcf2-4ca4-8f97-d827d3252f1b.png)

Maybe we should do `VERCEL_URL===https://www.cloudquery.io` or something, but for the sake of fixing previews I don't think it matters